### DISCUSSION TITLE
Problem with semantic highlighting

### DISCUSSION THREAD

**Discussion by Killer-Giga** [OmniSharp/omnisharp-vscode#7787] [Q&A]
I have a problem with vs code. 
When I change the theme the colors change and everything is fine until then but some colors change immediately after the change
For example I have Console.WriteLine(something);
Console is blue
WriteLine turns yellow and after a second it turns blue like Console (This is the problem I have)
(something) blue and it's fine like that.

When I disable the C# extension this problem is fixed. But I need C# in vs code.
What can I do?

![Captura de pantalla 2024-11-14 104702](https://github.com/user-attachments/assets/40a7f75a-2d0e-4cf9-af6e-33792d4aa49c)

**Comment by dibarbet** [comment]
Apologies for the delay here, generally if you're having a problem with the extension, please file an issue as we see those quicker.
@Killer-Giga generally if it is all blue like that, it means there was some problem loading the project.  A couple questions
1.  Do you have a project or solution file in your workspace?  The C# extension requires a project or solution file to run.
2.  Is the C# extension loading the project?  In the `C#` output window you should see a log line like `2024-12-04 17:32:28.950 [info] [Info  - 5:32:28 PM] [LanguageServerProjectSystem] Successfully completed load of c:\Users\dabarbet\source\repos\trash\ConsoleAppNet9\ConsoleAppNet9.csproj`
